app/src/main/java/chat/rocket/android/profile/presentation/ProfilePresenter.kt

```diff
@@ -221,4 +233,45 @@ class ProfilePresenter @Inject constructor(
             }
         }
     }
-}
\ No newline at end of file
+
+    // WIDECHAT
+    fun deleteAccount(username: String, ssoDeleteCallback: () -> Unit?) {
```

Comment:
would it be better practice to name this something other than the already existing deleteAccount? like deleteWidechatAccount?

Reply:
I'm ambivalent about which is the best practice in this case, since this function is essentially agnostic to widechat in that it's only different for including that ssoDeleteCallback, which any other SSO service could implement.  If you prefer the widechat function name I can do that.